src/miral/basic_window_manager.cpp

```diff
@@ -924,6 +929,11 @@ void miral::BasicWindowManager::modify_window(WindowInfo& window_info, WindowSpe
 
     Point new_pos = modifications.top_left().is_set() ? modifications.top_left().value() : window.top_left();
 
+    if (window_info.type() == mir_window_type_anchored && modifications.anchor_edge().is_set())
```

Comment:
If we can have `type()== mir_window_type_anchored` but not have the edge set, then maybe `mir_window_type_anchored` is a poor name?

Should it be `mir_window_type_layer`?